src/status_im/core.cljs

```diff
@@ -2,6 +2,9 @@
   (:require
     ;; NOTE: Do NOT sort i18n-resources because it MUST be loaded first.
     [status-im.setup.i18n-resources :as i18n-resources]
+   ;; NOTE: reagent.config Must be before any component definition,
+   ;; so they take the config properly
+    [reagent.config]
     #_{:clj-kondo/ignore [:unsorted-required-namespaces]}
```

Comment:
I need help to pass the lint checks for this require :( it's marked as unsorted.

cc: @ilmotta

Reply:
@ulisesmac To solve your problem, try adding `#_{:clj-kondo/ignore [:unsorted-required-namespaces]}` before `[reagent.config]`, similar to how it's done for requiring `@walletconnect/react-native-compat`.